packages/core/examples/webLoader/hardcodedMetaDataProvider.ts

```diff
@@ -0,0 +1,54 @@
+// Add hardcoded meta data provider for color images
+export default function hardcodedMetaDataProvider(type, imageId, imageIds) {
+  const colonIndex = imageId.indexOf(':');
+  const scheme = imageId.substring(0, colonIndex);
+  if (scheme !== 'web') return;
+
+  if (type === 'imagePixelModule') {
+    const imagePixelModule = {
+      pixelRepresentation: 0,
+      bitsAllocated: 24,
+      bitsStored: 24,
+      highBit: 24,
+      photometricInterpretation: 'RGB',
+      samplesPerPixel: 3,
+    };
+
+    return imagePixelModule;
+  } else if (type === 'generalSeriesModule') {
+    const generalSeriesModule = {
+      modality: 'SC',
+    };
+
+    return generalSeriesModule;
+  } else if (type === 'imagePlaneModule') {
+    const index = imageIds.indexOf(imageId);
+    // console.warn(index);
+    const imagePlaneModule = {
+      imageOrientationPatient: [1, 0, 0, 0, 1, 0],
+      imagePositionPatient: [0, 0, index * 5],
```

Comment:
Does this positioning have any effect when we're loading it as a stack?

I just tried running the example without passing `imagePositionPatient` and it seems to work ok?

If it is required, what would be the correct way to have multiple stacks of images with this loader?

Reply:
no it does not, I was about to write a volume example as well, that is why I put it there. You can just ignore it

Reply:
Ok thanks for the info :+1: